src/docs/sphinx/tutorials/step11/Tutorial.rst

```diff
@@ -0,0 +1,234 @@
+The computational domain is discretized by ``FE1``, which is defined in the ``NumericalMethods`` section. 
+The material is named as ``rock``, whose mechanical properties are specified in the ``Constitutive`` section.
+
+.. literalinclude:: ../../../../../examples/plasticity/DruckerPrager_Verification.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DP_WELLBORE_SOLVER -->
+  :end-before: <!-- SPHINX_DP_WELLBORE_SOLVER_END -->
+
+
+Constitutive model: defining material properties with constitutive laws
+-----------------------------------------------------------
+
+For this drained wellbore problem, we simulate the elastoplastic deformation caused by wellbore contraction.
+A homogeneous domain with one solid material is assumed, whose mechanical properties are specified in the ``Constitutive`` section: 
+
+.. literalinclude:: ../../../../../examples/plasticity/DruckerPrager_Verification.xml
+    :language: xml
+    :start-after: <!-- SPHINX_DP_WELLBORE_MATERIAL -->
+    :end-before: <!-- SPHINX_DP_WELLBORE_MATERIAL_END -->
+
+
+Recall that in the ``SolidMechanicsLagrangianSSLE`` section, 
+``rock`` is designated as the material in the computational domain. 
+Here, Extended Drucker Prager model ``ExtendedDruckerPrager`` is used to simulate the elastoplastic behavior of ``rock``.
+As for the material parameters, ``defaultInitialFrictionAngle``, ``defaultResidualFrictionAngle`` and ``defaultCohesion`` denote the initial friction angle, the residual friction angle, and cohesion, respectively, as defined by the Mohr-Coulomb failure envelope in the :math:`\Sigma` - :math:`\Tau` plane.
+As the residual friction angle ``defaultResidualFrictionAngle`` is larger than the initial one ``defaultInitialFrictionAngle``, the strain hardening model is automatically chosen, whose hardening rate is given as ``defaultHardening="0.01"``. 
+If the residual friction angle is set to be less than the initial one, strain weakening will take place. 
+And ``defaultDilationRatio="1.0"`` corresponds to an associated flow rule.
+The constitutive parameters such as the density, the bulk modulus, and the shear modulus are specified in the International System of Units.
+
+
+Initial and Boundary Conditions: defining properties with the FieldSpecifications
+-----------------------------------------------------------
+
+The next step is to specify fields, including:
+
+  - The initial value (the in-situ stresses and traction at the wellbore wall have to be initialized)
+  - The boundary conditions (the reduction of wellbore pressure and constraints of the outer boundaries have to be set)
+
+In this tutorial, we need to specify isotropic horizontal stress (:math:`\Sigma_h` = 11.25 MPa) and vertical stress (:math:`\Sigma_v` = 15.0 MPa). 
+To reach equilibrium, a compressive traction :math:`P_w` = -11.25 MPa is instantaneously applied at the wellbore wall ``rneg`` at time :math:`t` = 0 s, which will then be gradually reduced to a lower value (-2.0 MPa) to let wellbore contract.
```

Comment:
Perhaps add a note that the traction loading has a negative value due to the surface sign convention in GEOSX.